tasks.py

```diff
@@ -175,10 +175,11 @@ def run_command(context, command, **kwargs):
         # Check if Nautobot is running; no need to start another Nautobot container to run a command
         docker_compose_status = "ps --services --filter status=running"
         results = docker_compose(context, docker_compose_status, hide="out")
+        docker_container = kwargs.pop("container", "nautobot")
```

Comment:
I _think_ the better approach would to make this a named argument in the function signature with a default?